js/index.js

```diff
@@ -60,6 +60,19 @@ function populateIngredients(recipe) {
   document.getElementById("ingredient-single-view").innerHTML = list;
 };
 
+
+
+function createDropDown()
+{
+    var currentLab = document.getElementsByClassName("lab-selected")[0].innerHTML;
+    var dropDownHTML = "";
+    dropDownHTML += "<option value='first'>" + allRecipes[currentLab][0].name + "</option>";
+    dropDownHTML += "<option value='second'>" + allRecipes[currentLab][1].name + "</option>";
+    dropDownHTML += "<option value='third'>" + allRecipes[currentLab][2].name + "</option>";
+    dropDownHTML += "<option value='fourth'>" + allRecipes[currentLab][3].name + "</option>";
+    document.getElementById("drop-down-list").innerHTML = dropDownHTML;
+}
```

Comment:
As a bonus, if you switch to using array indicies, this code becomes a for loop instead! Woo!

    for (var i=0; i < allRecipes[currentLab].length; ++i) {
        dropDownHTML += "<option value='" + i + "'>" + allRecipes[currentLab][i].name + "</option>";
    }